mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(priceNoTax) {
```

Comment:
Thought: What you've done is exactly right, however I think the function name you've been given is misleading as it suggests it will return the sales tax only and not the price with sales tax added. Maybe a better name would be `calculateAndAddSalesTax`